airflow/configuration.py

```diff
@@ -159,11 +159,6 @@ class AirflowConfigParser(ConfigParser):
             'task_runner': ('BashTaskRunner', 'StandardTaskRunner', '2.0'),
         },
     }
-    deprecation_value_format_string = (
-        'The {name} setting in [{section}] has the old default value of {old!r}. This '
-        'value has been changed to {new!r} in the running config, but please '
-        'update your config before Apache Airflow {version}.'
-    )
```

Comment:
Do the same for `deprecation_format_string` too? (it didn't use locals() I suspect)

Reply:
Done